R/tbl_split.R

```diff
@@ -65,3 +147,62 @@ print.tbl_split <- function(x, ...) {
   check_dots_empty()
   walk(x, print)
 }
```

Comment:
how do you want to deal with this? make the viewer do a side by side print? I would do it as an option maybe?

Reply:
I would keep it like this for now